frameworks/g-cloud-9/questions/services/installationCompatibleOperatingSystems.yml

```diff
@@ -1,20 +1,28 @@
 name: Compatible operating systems
 question: Which operation systems does your service work with?
 
+hidden: true
 depends:
   - "on": lot
     being:
       - cloud-software
 
 type: checkboxes
 options:
+  - label: Android
+    value: android
   - label: iOS
+    value: ios
   - label: Linux or Unix
+    value: linux
```

Comment:
You can't have `linux` here and `unix` on the other one - they should be the same value for the same "Linux or Unix" label.